connect/support/encoding.py

```diff
@@ -30,6 +33,8 @@ def default(self, o: Any) -> Any:
             return str(o)
         elif isinstance(o, bytes):
             return base64.b64encode(o).decode()
+        elif isinstance(o, decimal.Decimal):
```

Comment:
It may be preferable to return str(o)
```
>>> d = Decimal("10.123131")
>>> str(d)
'10.123131'
```
This will return the string representation of the Decimal without converting it to a float. That being said the Decimal __init__ accepts a string, int, or float, so I think it would still work.

Reply:
float() allows the object to remain a number, because once you convert it to str(), you can't parse it back to a number.